app/src/info/WorldDataInfoGenerator.ts

```diff
@@ -30,40 +50,23 @@ export default class WorldDataInfoGenerator implements IProjectInfoItemGenerator
   performPlatformVersionValidations: boolean = false;
 
   getTopicData(topicId: number) {
-    switch (topicId) {
-      case 1:
-        return { title: "Block" };
-      case 2:
-        return { title: "Block Data" };
-      case 3:
-        return { title: "Command" };
-      case 4:
-        return { title: "Execute Sub Command" };
-      case 5:
-        return { title: "Level.dat" };
-      case 6:
-        return { title: "Level.dat Experiments" };
-      case 7:
-        return { title: "Subchunkless Chunks" };
-      case 101:
-        return { title: "Unexpected command in MCFunction" };
-      case 102:
-        return { title: "Unexpected command in Command Block" };
-      case 400:
-        return { title: "Error processing world" };
-    }
-
     return {
-      title: topicId.toString(),
+      title: ProjectInfoUtilities.getTitleFromEnum(WorldDataInfoGeneratorTest, topicId),
     };
   }
 
   summarize(info: any, infoSet: ProjectInfoSet) {
-    info.chunkCount = infoSet.getSummedNumberValue("WORLDDATA", 101);
+    info.chunkCount = infoSet.getSummedNumberValue(
+      "WORLDDATA",
+      WorldDataInfoGeneratorTest.unexpectedCommandInMCFunction
+    );
 
-    info.subchunkLessChunkCount = infoSet.getSummedNumberValue("WORLDDATA", 107);
+    info.subchunkLessChunkCount = infoSet.getSummedNumberValue(
+      "WORLDDATA",
+      WorldDataInfoGeneratorTest.subchunklessChunks
```

Comment:
`WorldDataInfoGeneratorTest.subchunklessChunks` == 7, whereas before we used 107. Missing value in the enum?

Reply:
Yeah, I think this was an error originally in the code (in one place it was 7 and in another it was 107) that converting to enums helped me to catch